### DISCUSSION TITLE
Apparent or Real Power?

### DISCUSSION THREAD

**Discussion by henfri** [jertel/vuegraf#136] [Q&A] [answered]
Hello,

Does the device output apparent power (Current*Voltage) or true/real power (Current*Voltage*cos(phase shift))?

Best regards,
Hendrik

**Comment by jertel** [comment, marked as answer]
I believe the device is measuring both apparent and real power, but the app (and Vuegraf) shows only real power.

Some non-US users with 3-phase systems have complained (search the Emporia forums) about discrepancies in this area, specifically that the branch CTs appear to be measuring apparent power instead of real power, while the main CTs are measuring real power.

**Reply by henfri** [reply]
Thank you. I read in some cases the claim that apparent power is measured.
But on the other hand, I would expect a lot more complaints, as especially at low power (standby) PSUs can show 10 times more apparent power than real power.
I am surprised, no-one has measured it yet.
